controllers/transfer.rb

```diff
@@ -0,0 +1,73 @@
+class CustomWizard::TransferController < ::ApplicationController
+  before_action :ensure_logged_in
+  before_action :ensure_admin
+
+  skip_before_action :check_xhr, :only => [:export]
+
+
+  def index
+
+  end
+
+  def export
+
+    wizards = params['wizards']
+    wizard_objects = []
+    wizards.each do
+    |w|
+      # p w
+
+      wizard_objects.push(PluginStore.get('custom_wizard', w.tr('-', '_')))
+
+    end
+    puts 'wizard_objects'
+    p wizard_objects
+    send_data wizard_objects.to_json, type: "application/json", disposition: 'attachment', filename: 'wizards.json'
+
+  end
+
+  def is_json(string)
+    begin
+      !!JSON.parse(string)
+    rescue
+      false
+    end
+  end
+
+  def import
+    file = File.read(params['file'].tempfile)
+    fileSize = file.size
+    maxFileSize = 512 * 1024
+    if maxFileSize < fileSize
+      render json: {error: "File too large"}
```

Comment:
Let's pop these errors into the locales/server.en.yml file, maybe under wizard.import.error